apps/fyle/tasks.py

```
@@ -712,7 +712,9 @@ def update_non_exported_expenses(data: Dict) -> None:
             expense_objects = FyleExpenses().construct_expense_object(expense_obj, expense.workspace_id)
 
             old_fund_source = expense.fund_source
+            old_category = expense.category if (expense.category == expense.sub_category or expense.sub_category == None) else '{0} / {1}'.format(expense.category, expense.sub_category)
```

Flake8 (7.3.0)

[error] comparison to None should be 'if cond is None:'

(E711)